app/controllers/concerns/setup_repo.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module SetupRepo
+  extend ActiveSupport::Concern
+  IMPORT_IN_PROGRESS     = "Importing starter code to student repo"
+  CONFIGURATION_PROGRESS = "Configuring the student repo"
+
+  def setup_status(repo, config)
+    progress = { status: "importing", message: IMPORT_IN_PROGRESS }
```

Comment:
If the status is only being used for comparisons, these should be either symbols or strings referenced by a constant.